processor/blockprocessor/initialize.go

```diff
@@ -175,7 +175,7 @@ func blockHandler(logger *log.Logger, dbRound uint64, proc processor.Processor,
 			case <-ctx.Done():
 				return err
 			case <-time.After(retryDelay):
-				break
+				// noop
```

Comment:
If this change is right (it looks like it is, but what do I know?), it seems important enough to do in another PR now, rather than wait for boxes.

Reply:
I made a [suggestion](https://github.com/algorand/indexer/pull/1085#pullrequestreview-1026432861) for the main localledger pr

Reply:
Suggestion was taken. Resolving.